polyfills/Object/values/polyfill.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+	function values(object) {
+		return Object.keys(object).map(function (key) {
+			return object[key];
+		});
+	}
+
+	Object.defineProperty(Object, 'values', {
+		configurable: true,
+		enumerate: false,
```

Comment:
`enumerate` is not a valid object descriptor property.  Should be [`enumerable`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#Description).  It is already false by default, but technically required, so we should include it.